### DISCUSSION TITLE
Is it possible to simulate Ethereum The Merge with Ganache ?

### DISCUSSION THREAD

**Discussion by mksh** [ConsenSys-archive/truffle#5480] [General Q&A] [answered]
Hello Trufflesuite!

It seems that it is not possible to simulate Ethereum The Merge happening, with Ganache as execution client. When I configure terminal total difficulty to some reachable time, the concensus clients attempts to invoke `engine_exchangeTransitionConfigurationV1` method as described in [EIP-3675](https://github.com/ethereum/execution-apis/blob/v1.0.0-alpha.9/src/engine/specification.md#engine_exchangetransitionconfigurationv1). But, Ganache responds with an error that is shown below:

```
code: -32700, message: "The method engine_exchangeTransitionConfigurationV1 does not exist/is not available
```

I have a question -- are there any plans to support The Merge going forward ? Can we expect this to be done before  The Merge Mainnet ?

Thank you in advance!

**Comment by apdan510** [comment, marked as answer]
Hey @mksh 

A merge specific version of Ganache is not quite ready yet for release, but it is coming very soon. Please stay tuned and thank you!
